packages/gatsby-link/src/index.js

```diff
@@ -1,4 +1,4 @@
-/*global __PATH_PREFIX__ */
+/*global __BASE_PATH__ */
```

Comment:
path prefix is also global?

```suggestion
/*global __BASE_PATH__ */
/*global __PATH_PREFIX__ */
```

Reply:
These aren't really needed anyways. I'll just remove the comment.

Reply:
Also - none of these are used in this file--so this comment was a little misplaced to begin with!